auth-web/src/models/affiliation.ts

```diff
@@ -66,9 +66,11 @@ export interface NameRequestResponse {
   natureOfBusiness?: string
   expirationDate?: Date
   nrNum?: string
+  requestActionCd?: string
   stateCd?: string
   natureBusinessInfo?: string
   applicants?: Array<Applicant>
+  corpNum?: string
```

Comment:
These are the new fields coming from Namex, right?

Reply:
Yes 😃